### DISCUSSION TITLE
en/chapter_hashing/hash_collision/

### DISCUSSION THREAD

**Discussion by giscus** [krahets/hello-algo#1592] [Announcements]
# en/chapter_hashing/hash_collision/

Data Structures and Algorithms Crash Course with Animated Illustrations and Off-the-Shelf Code

https://www.hello-algo.com/en/chapter_hashing/hash_collision/

**Comment by SantiDu** [comment]
In double hashing, what to do when all spaces are occupied? If the problem is not addressed, the algorithm goes into an infinite loop.
